processor/src/it/java/com/linecorp/decaton/processor/RetryQueueingTest.java

```diff
@@ -89,6 +92,8 @@ public void testRetryQueuing() throws Exception {
                 .retryConfig(RetryConfig.builder()
                                         .retryTopic(retryTopic)
                                         .backoff(Duration.ofMillis(10))
+                                        .producerSupplier(config -> new TestTracingProducer(
```

Comment:
The retry producer needs to propagate the trace id, since the test asserts that it's set correctly at the point where the message is processed (even on retry). In the real case a zipkin tracing kafka producer will do this (I put a note about this in the comment on `TracingProvider`).

Reply:
I see. Please check that in to-be added tracing integration test rather than modifying existing RetryQueueingTest.